classes/class-wc-rest-connect-shipping-carrier-controller.php

```diff
@@ -26,6 +26,7 @@ public function post( $request ) {
 			return $error;
 		}
 
+		do_action( 'wc_connect_fetch_service_schemas' );
```

Comment:
@superdav42, I just added this line to refresh the shipping methods regardless of the value for `WOOCOMMERCE_CONNECT_FREQUENT_FETCH`. Seems a bit too easy: will this do the trick or would you recommend a better way to call `WC_Connect_Service_Schemas_Store::fetch_service_schemas_from_connect_server`?

Reply:
Using the action is perfect. You just need it add it to `WC_REST_Connect_Shipping_Carrier_Delete_Controller::delete()` as well.